recipes/sonic-cpp/all/conanfile.py

```diff
@@ -0,0 +1,77 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.microsoft import is_msvc
+from conan.tools.scm import Version
+import os
+
+required_conan_version = ">=1.53.0"
+
+class SonicCppConan(ConanFile):
+    name = "sonic-cpp"
+    description = "A fast JSON serializing & deserializing library, accelerated by SIMD."
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/bytedance/sonic-cpp"
+    topics = ("json", "parser", "writer", "serializer", "deserializer", "header-only")
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    @property
+    def _min_cppstd(self):
+        return 11
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def layout(self):
+        basic_layout(self, src_folder="src")
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+			"gcc": "8",
+			"clang": "7",
+			"apple-clang": "12",
+		}
```

Comment:
```suggestion
        return {
            "gcc": "8",
            "clang": "7",
            "apple-clang": "12",
        }
```

avoid mixing tab and spaces. python interpreter will not break, but will warn you because is not recommended by PEP8.